models/README.md

```diff
@@ -7,48 +7,41 @@ in this directory. Please also specify most recent **validation loss** and numbe
 * ``sdss_autoencoder.ckpt``
 
 ```bash
-# Validation Loss = 781677.125
-# Epochs = 969
-poetry run bliss mode="train" model="galaxy_net" dataset="sdss_galaxies" optimizer="adam" \
-training.n_epochs=1001 training.trainer.check_val_every_n_epoch=10 dataset.kwargs.noise_factor=0.01 \
-dataset.kwargs.num_workers=5 training.trainer.checkpoint_callback=True
+# Validation Loss = 782252.688
+# Epochs = 999
+poetry run bliss +experiment=sdss_autoencoder
 ```
 
 * ``sdss_binary.ckpt``
 
 ```bash
-# Validation Loss = 6.571
-# Epochs = 359
+# Validation Loss = 7.472
+# Epochs = 439
 # Notes:
 # - 20/09/21: Slight increase since galaxy PSF exactly equals decoder PSF
-poetry run bliss mode="train" model="binary_sdss" dataset="default" \
-optimizer.kwargs.lr=1e-4 training.n_epochs=501 training.trainer.checkpoint_callback=True \
-model.decoder.kwargs.mean_sources=0.03 training.save_top_k=5
+# - 20/09/29: Slightly worse than the previous one (0.571)
+poetry run bliss +experiment=sdss_binary
 ```
 
 * ``sdss_galaxy_encoder.ckpt``
 
 ```bash
-# Validation Loss = 1295313.875
-# Epochs = 1174
+# Validation Loss = 1293265.625
+# Epochs = 1849
 # Notes:
 # - 20/09/21: Training for even longer might be possible.
-poetry run bliss mode="train" model="galenc_sdss" dataset="default" \
-optimizer.kwargs.lr=1e-4 training.n_epochs=2001 training.trainer.checkpoint_callback=True \
-model.kwargs.decoder_kwargs.mean_sources=0.04 training.trainer.check_val_every_n_epoch=25
+poetry run bliss +experiment=sdss_galaxy_encoder
 ```
 
 * ``sdss_sleep.ckpt``
 
 ```bash
-# Validation Loss = -0.102
-# Epochs = 1349
+# Validation Loss = -0.042
+# Epochs = 1499
```

Comment:
This is significantly higher than -0.102, are you rerunning it now?

Reply:
I tried one more time and it was -0.044. Does this look normal and we should just retrain it with #323, or does it suggesting something bigger?

Reply:
That is suspicious, let me reran the model on master and see what happens

Reply:
Hi Ziteng, thanks for checking this. I find this strange so I will run this in the master branch and investigate this further